Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/MotionAlertDialog.java

```diff
@@ -60,117 +51,100 @@ public boolean onKeyDown(int keyCode, KeyEvent event)
 		}
 	}
 
+	@Override
+	public boolean dispatchKeyEvent(KeyEvent event)
+	{
+		// Handle this key if we care about it, otherwise pass it down the framework
+		return onKeyEvent(event.getKeyCode(), event) || super.dispatchKeyEvent(event);
+	}
+
+	@Override
+	public boolean dispatchGenericMotionEvent(MotionEvent event)
+	{
+		// Handle this event if we care about it, otherwise pass it down the framework
+		return onMotionEvent(event) || super.dispatchGenericMotionEvent(event);
+	}
 
-	// Method that will be called within dispatchGenericMotionEvent
-	// that handles joystick/controller movements.
 	private boolean onMotionEvent(MotionEvent event)
 	{
-		if ((event.getSource() & InputDevice.SOURCE_CLASS_JOYSTICK) == 0)
+		if (((event.getSource() & InputDevice.SOURCE_CLASS_JOYSTICK) == 0))
```

Comment:
Why is an extra set of parentheses added here?

Reply:
It was from a prior change, fixed.